tests/tools/test_dist_part.py

```diff
@@ -8,120 +8,53 @@
 from dgl.data.utils import load_tensors, load_graphs
 from chunk_graph import chunk_graph
 
+from create_chunked_dataset import create_chunked_dataset
+
+'''
 @pytest.mark.parametrize("num_chunks", [1, 2, 3, 4, 8])
 @pytest.mark.parametrize("num_parts", [1, 2, 3, 4, 8])
+'''
+@pytest.mark.parametrize("num_chunks", [2])
```

Comment:
even in the simplest case, the CI failed yet. pls fix it.
And could we enable all the combinations of `num_chunks` and `num_parts`. and if `num_chunks!=num_parts`, skip the edge feature assignment and verification?

Reply:
I have spent a lot of debugging this issue and realized that this is because of merge conflict resolution and some misleading comments made by Minjie that orig-ids are saved in separate files and NOT as node features. But the latest master code looks completely opposite. The latest master code treats the orig-ids as both node-feautres and stores them separately as "dgl" files. 

I made these above changes to 'create_chunked_dataset.py' and test_dist_part.py and after these changes this issue seems to be fixed and its working. 

I will test it with edge features and see if it works, otherwise I will revert back to you for possible steps moving forward. 

Needless to say, the values for the input parameters to this function will be restored to its original form, once this debugging is completed.